utilities/after_success.sh

```diff
@@ -31,6 +31,8 @@ if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -
         # Update versions README and pom.xml in master branch
         cd ..
         utilities/update_docs_version.sh
+
```

Comment:
I didn't want to associate `mvn deploy` with the comment at the top of the preceding block (update readme versions), but I can make that comment an inline comment and remove this extra line.  I don't feel strongly about the line either way.